src/amg/azext_amg/commands.py

```diff
@@ -0,0 +1,45 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+
+from ._validators import process_grafana_create_namespace
+
+
+def load_command_table(self, _):
+
+    with self.command_group('grafana', is_preview=True) as g:
+        g.custom_command('create', 'create_grafana', validator=process_grafana_create_namespace)
+        g.custom_command('delete', 'delete_grafana')
```

Comment:
You can use `confirmation=True` which will add `--yes` argument by azure-cli-core framework and prop user confirmation when run delete command.
 
https://github.com/Azure/azure-cli-extensions/blob/943d81d738ee3bc6283b59fba98860338fb2da98/src/network-manager/azext_network_manager/commands.py#L72